src/frontends/onnx/frontend/src/op/celu.cpp

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include "op/celu.hpp"
+
+#include <memory>
+
+#include "default_opset.hpp"
+#include "exceptions.hpp"
+#include "utils/common.hpp"
+
+OPENVINO_SUPPRESS_DEPRECATED_START
+namespace ngraph {
+namespace onnx_import {
+namespace op {
+namespace set_1 {
+OutputVector celu(const Node& node) {
+    double alpha = node.get_attribute_value<float>("alpha", 1.0);
+    auto alpha_node = default_opset::Constant::create(element::f32, Shape{}, {alpha});
+    auto zero_node = default_opset::Constant::create(element::f32, Shape{}, {0});
+    auto one_node = default_opset::Constant::create(element::f32, Shape{}, {1});
+    auto input = node.get_ng_inputs().at(0);
+    CHECK_VALID_NODE(node, input.get_element_type() == element::f32, "Only float32 input type is supported")
```

Comment:
Please remove this check, supported types are validated in the constructors of ov operators, also such restriction can be not backward compatible.